config/stemmer.yml

```diff
@@ -26,6 +26,9 @@ step_1_exceptions:
   ΚΡΕΑΤΟΣ: ΚΡΕ
   ΚΡΕΑΤΑ: ΚΡΕ
   ΚΡΕΑΤΩΝ: ΚΡΕ
+  ΝΤΟΥΛΑΠΑΣ: ΝΤΟΥΛΑΠΑ
+  ΝΤΟΥΛΑΠΩΝ: ΝΤΟΥΛΑΠΑ
```

Comment:
For this case and the family of cases under `ΧΑΡΤΗΣ`, it is preferable if we handle them with code instead of hard-coded, so that this file remains identical to the one of the java version.